### DISCUSSION TITLE
[archived] "Debug Pause" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10403] [General]
# argo-workflows/debug-pause/

https://argoproj.github.io/argo-workflows/debug-pause/

**Comment by jrhoward** [comment]
this is gold !!

**Comment by kalb1** [comment]
For those poor guys who have to use it on windows pod:
Powershell command:
```powershell
New-Item C:\var\run\argo\ctr\main\{after,before}
```
